misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/SpecBodyExtensions.kt

```diff
@@ -1,26 +1,29 @@
-//TODO remove file as soon as all specs have been transformed to spek2
-package ch.tutteli.atrium.specs
+// //TODO remove file as soon as all specs have been transformed to spek2
+// package ch.tutteli.atrium.specs
```

Comment:
You commented the code out and the build is still green, ergo you can remove the file :slightly_smiling_face: